wpeverest/user-registration
# UR-463 Dev - Compatibility for SMS Verification with Twilio Integration

### All Submissions:






### Changes proposed in this Pull Request:

Introducing new module called SMS Integration, which allows you to integrate Twilio account for SMS Verification. It will also create new login option called sms-verification. 

### How to test the changes in this Pull Request:

1. Create Twilio account and connect from the Settings->Integration->Twilio.
2. Create a form with SMS Verification login option and Map Smart Phone Field for SMS Verification. Make sure Mapped fields should be required.
3. Create a user with that form and then goto login page and try to login. 
4. It will redirect you to OTP page. It will send sms with OTP in your mapped mobile number.
5. Now you can resend code or verify otp. Once OTP verified user will allow to login and it's status will be Approved.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Compatibility for SMS Verification with Twilio Integration
